### ISSUE TITLE
wear thoughts

### ISSUE THREAD

**Issue by SwissalpS** [SwissalpS/replacer#49]
see: https://github.com/pandorabox-io/in-game/issues/327

**Comment by SwissalpS** [comment]
BTW, stumbled upon this: https://github.com/SwissalpS/replacer/commit/56f8dfe1aabf28231a02ae1f629b86d700b2597c
08:38 < SwissalpS> that's why I was sure I had disabled water rotation.
08:47 < pandorabot> [minetest] <Huhhila> so that deny list would deny both as source and destination material, while only denying as replacement/rotation would be needed?
08:53 < SwissalpS> or a special handler that only allows rotating to certain param2. That's something a fully re-written replacer would be able to do.
08:54 < pandorabot> [minetest] <Huhhila> yup capping it at some balanced number would be nice :) Maybe even use more of the replacer  charge for higher values ;)

TL;DR: new replacer needs callback registration so other mods can modify per-situation results of param2 and charge usage. Eliminating the need for the deny-list.
